### ISSUE TITLE
stale bounties filter

### ISSUE THREAD

**Issue by owocki** [gitcoinco/web#197]
put a radio button on the gitcoin.co/explorer sidebar that filters out any bounties that have had action on them in the last X days.  these bounties are *stale* in that whomever has claimed them is not doing active work

X = 5 for intiial MVP

**Comment by owocki** [comment]
cc @vs77bb in case this would help you in your weekly pruning of bounties in any way...

**Comment by gitcoinbot** [comment]
__This issue now has a funding of 0.1 ETH (54.11 USD @ $541.12/ETH) attached to it.__

 * If you would like to work on this issue you can claim it [here](https://gitcoin.co/issue/gitcoinco/web/197).
 * If you've completed this issue and want to claim the bounty you can do so [here](https://gitcoin.co/issue/gitcoinco/web/197)
 * Questions? Get help on the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $3038.65 more Funded OSS Work Available at: https://gitcoin.co/explorer

**Comment by gitcoinbot** [comment]
__Work has been started on the 0.1 ETH (55.29 USD @ $552.89/ETH) funding by__: 
 1. [@leonj3813](https://gitcoin.co/profile/leonj3813) 

 __Please work together__ and coordinate delivery of the issue scope. Gitcoin doesn't know enough about everyones skillsets / free time to say who should work on what, but we trust that the community is smart and well-intentioned enough to work together.  As a general rule; if you start work first, youll be at the top of the above list ^^, and should have 'dibs' as long as you follow through. 

 On the above list? Please leave a comment to let the funder (@owocki) and the other parties involved what you're working, with respect to this issue and your plans to resolve it.  If you don't leave a comment, the funder may expire your submission at their discretion. 

 * Learn more [on the gitcoin issue page](https://gitcoin.co/issue/gitcoinco/web/197)
 * Questions? Get help on the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $3314.86 more Funded OSS Work Available at: https://gitcoin.co/explorer

**Comment by vs77bb** [comment]
Hi @leonj3813 are you working this one right now? I think this is a great task that can help us improve the product experience a ton 🙂

**Comment by leonj3813** [comment]
Hey @vs77bb I'm working on adding this. I'm setting up my local environment and then I'll add to the refreshBounties uri to get the new information from the HTTPS API.

**Comment by owocki** [comment]
> I can filter x days from the time in the "modified_on" field easy enough.

this works!

**Comment by leonj3813** [comment]
@owocki Yeah I tested this last night and it looks like the modified_on field is getting updated every 15 minutes on the live api https://gitcoin.co/api/v0.1/bounties/. 
Maybe a cron job is running that runs refresh_bounties.py every 15 minutes to update the github comment count, then this saves the model which updates the modified_on field, I don't know for sure. 

If this is the case then I guess a last_commented_date field would have to be added to the model and this could be updated in refresh_bounties.py. This field would probably help with #567 also.

**Comment by owocki** [comment]
@leonj3813 ed08a04a7a0e5730751fa196d8fb48355f5dba5c should give u what u need

**Comment by vs77bb** [comment]
@owocki Is this one with you for review?

**Comment by owocki** [comment]
from [the pr](https://github.com/gitcoinco/web/pull/700) we are waiting on the turnaround of a few comments from @leonj3813 -- then we can pay it out

**Comment by gitcoinbot** [comment]
@davidbanu are you still working on this issue?

**Comment by gitcoinbot** [comment]
@davidbanu are you still working on this issue?

**Comment by gitcoinbot** [comment]
@davidbanu are you still working on this issue?

**Comment by gitcoinbot** [comment]
@davidbanu are you still working on this issue?
* [x] warning 1 (3 days)
* [x] warning 2 (6 days)
* [ ] auto removal (10 days)

**Comment by gitcoinbot** [comment]
@davidbanu Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] warning (3 days)
* [ ] auto removal (6 days)

**Comment by gitcoinbot** [comment]
@davidbanu Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] warning (3 days)
* [ ] auto removal (6 days)

**Comment by gitcoinbot** [comment]
@davidbanu Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] warning (3 days)
* [ ] auto removal (6 days)

**Comment by vs77bb** [comment]
@owocki @mbeacom I think we don't need this filter anymore given our new plan for managing stale bounties elsewhere. @owocki you can probably claw back funds here

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Cancelled** 

<hr>

__The funding of 0.1 ETH (74.1 USD @ $741.02/ETH) attached to this issue has been **cancelled** by the bounty submitter__

 * Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $8,466.90 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by thelostone-mc** [comment]
>  I think we don't need this filter anymore given our new plan for managing stale bounties elsewhere

Closing this!

**Comment by weberV** [comment]
Issue Status: 1. Open 2. **Cancelled** 

<hr>

__This issue now has a funding of 0.1 ETH (69.21 USD @ $692.1/ETH) attached to it.__

 * If you would like to work on this issue you can 'start work' [on the Gitcoin Issue Details page](http://localhost:8000/issue/gitcoinco/web/197/165).
 * Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $1,841.03 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)
